301-custom-images-at-scale/README.md

```diff
@@ -0,0 +1,9 @@
+# Custom Images at Scale
```

Comment:
https://github.com/Azure/azure-quickstart-templates/tree/master/mongodb-high-availability

Take a look at this template and see how various different template files are named. shared-resources.json etc.